server/controllers/auth/login.js

```diff
@@ -0,0 +1,25 @@
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const user = require('../../database/models/applicant');
+
+const SECRET = 'aplecant';
```

Comment:
This review is only for typo errors, when you correct them all and solve the conflicts, then we will start in code review 
```suggestion
const SECRET = 'applicant';
```